src/actions.js

```diff
@@ -10,7 +10,9 @@ module.exports = {
 
 	startGame: function(){
 		return function(dispatch, getState){
-
+				//clear the bloodsplatters
+				getState().corpses = {};
```

Comment:
Note that mutating state in an action creator is considered very bad form. Sort of like using the wrong cutlery to the wrong dish at the Nobel dinner! This should be done in the reducer.

But in this case you can blame the practice already established by @as223jx! ;)